### PR TITLE
Added region mapping for IMAGE 3.3

### PR DESCRIPTION
**Pull request by VassilisDaioglou**

### CODE REVIEW

**Comment by danielhuppmann** [comment]
@VassilisDaioglou, are you planning to use this for ECEMF?

**Comment by VassilisDaioglou** [comment]
@danielhuppmann Yes, this should also be used for ECEMF to be fully consistent with our scenario uploads.

File: definitions/region/model_native_regions/image_v3.3.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# The model documentation and regional aggregation for IMAGE (TIMER) version 3.2 can be found at https://models.pbl.nl/image/index.php/Region_classification_map
+- IMAGE 3.2:
```
Comment (danielhuppmann) [inline definitions/region/model_native_regions/image_v3.3.yaml:2]: Also in the lines below...
```suggestion
- IMAGE 3.3:
```

**Review by danielhuppmann — APPROVED** [review]
Looks good, thanks!

**Pull request merged**